.github/workflows/Ci.yml

```diff
@@ -6,7 +6,19 @@ on:
   push:
     branches: [develop, master]
 jobs:
+  # First job: Preliminary setup or determinations
+  preliminary:

```

Comment:
Can we update this step to be named the same way it is named in other branches

Reply:
https://github.com/usdot-fhwa-stol/carma-messenger/blob/76b2b7436d46ef64b67ff2fa362f2c802d51fa6c/.github/workflows/ci.yml#L11

Reply:
remamed